src/uwtools/api/config.py

```diff
@@ -159,7 +158,7 @@ def realize_to_dict(  # pylint: disable=unused-argument
 
     See ``realize()`` for details on arguments, etc.
     """
-    return _realize(**{**locals(), "output_file": Path(os.devnull), "output_format": None})
+    return realize(**{**locals(), "output_file": Path(os.devnull), "output_format": _FORMAT.yaml})
```

Comment:
Note that
1. `key_path` and `total` are now accepted by `realize_to_dict()` as well, and passed through to `realize()`.
2. The output format is set to YAML because _some_ format needs to be available later in the library code, and YAML is an appropriate, generic format to associate with arbitrary `dict` objects, as there are no depth etc. concerns.